src/test/resources/pages/instructorFeedbackQuestionEditToTeamToTeam.html

```diff
@@ -974,42 +974,38 @@ <h4 class="modal-title">
                     <input checked="" class="visibilityCheckbox recipientCheckbox1" disabled="" name="receiverFollowerCheckbox" type="checkbox" value="RECEIVER">
                   </td>
                 </tr>
-                <hide style="display: none;">
-                  <tr>
-                    <td class="text-left">
-                      <div data-original-title="Control what team members of feedback giver can view" data-placement="top" data-toggle="tooltip" title="">
-                        Giver's Team Members
-                      </div>
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox answerCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox giverCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox recipientCheckbox1" type="checkbox" value="OWN_TEAM_MEMBERS">
-                    </td>
-                  </tr>
-                </hide>
-                <hide style="display: none;">
-                  <tr>
-                    <td class="text-left">
-                      <div data-original-title="Control what team members of feedback recipients can view" data-placement="top" data-toggle="tooltip" title="">
-                        Recipient's Team Members
-                      </div>
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox answerCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox giverCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox recipientCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                    </td>
-                  </tr>
-                </hide>
+                <tr>
+                  <td class="text-left">
```

Comment:
Yup, this should have been caught beforehand. @YongJieYongJie @JosephineKwa next time be careful, okay?